shell/components/nav/TopLevelMenu.vue

```diff
@@ -536,6 +536,8 @@ export default {
           >
             <nuxt-link
               :to="{ name: 'about' }"
+              class="version"
+              data-testid="footer__about_link"
```

Comment:
we have a few different concepts of footer in the dashboard. this will need to be more unique in case of clashes, or something else can be used to select the element

Reply:
removed it all together and used the` "side-menu"` to select the element